tests/View/Mix/MixTest.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Illuminate\Tests\View\Mix;
+
+use PHPUnit\Framework\TestCase;
+use Illuminate\Support\HtmlString;
+
+class MixTest extends TestCase
+{
+    public function setUp()
+    {
+        app()->singleton('path.public', function () {
+            return __DIR__;
+        });
+    }
+
+    private function getMix()
+    {
+        return new \Illuminate\View\Mix\Mix;
+    }
+
+    /**
+     * @expectedException \Illuminate\View\Mix\MixException
+     * @expectedExceptionMessage The Mix manifest does not exist.
+     */
+    public function testMixMethodThrowAnExceptionIfMixManifestDoesNotExist()
+    {
+        $this->getMix()->mix('foo.css');
+    }
+
+    /**
+     * @expectedException \Illuminate\View\Mix\MixException
+     * @expectedExceptionMessage Unable to locate Mix file: /baz.css. Please check your webpack.mix.js output paths and try again.
+     */
+    public function testMixMethodThrowAnExceptionIfPathDoesNotExistInManifest()
+    {
+        $this->getMix()->mix('baz.css', 'fixtures');
+    }
+
+    /**
+     * @expectedException \Illuminate\View\Mix\MixException
+     * @expectedExceptionMessage The Mix manifest isn't a proper json file.
+     */
+    public function testMixMethodThrowAnExceptionIfManifestIsNotAProperJson()
+    {
+        $this->getMix()->setManifestFilename('mix-manifest-wrong')->mix('foo.css', 'fixtures');
+    }
+
+    public function testMixMethodWhenDisabled()
+    {
+        $this->assertEquals(new HTMLString('Mix is disabled!'), $this->getMix()->disable()->mix('foo.css'));
+    }
+
+    /**
+     * @expectedException \Illuminate\View\Mix\MixException
+     */
```

Comment:
Ok, we don't suceed to understand each other, I'm just switching in our mutual native language, it will be easier for us. 😕

Ici on s'en fout de savoir ce qui arrive après, on veut juste s'assurer que la méthode `enable` "marche" comme il faut. Dès l'instant où il y a autre chose que le comportement de `disable`, donc une exception, peu importe laquelle dans ce cas là, ça veut dire que ça marche.

On peut rajouter le test du message, mais c'est une question de logique ici, on ne cherche pas à savoir quel message va être affiché, juste s'assurer que la fonction continue, et ne s'arrête pas sur le disable.